src/dither.cpp

```diff
@@ -34,43 +34,41 @@
 
 // Original version: Kevin Watts <watts@willowgarage.com>
 
-#include <control_toolbox/dither.h>
-
+#include <limits>
 #include <random>
 
-namespace control_toolbox {
+#include "control_toolbox/dither.hpp"
 
-Dither::Dither() : amplitude_(0), has_saved_value_(false)
+namespace control_toolbox
 {
+Dither::Dither()
+: amplitude_(0), has_saved_value_(false) {}
 
-}
-
-Dither::~Dither()
-{
-}
+Dither::~Dither() {}
```

Comment:
![20120514__dictator21](https://user-images.githubusercontent.com/3524577/75329006-d0c24800-5876-11ea-9de5-b55aa9f8852d.jpg)

Reply:
It's done.